src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java

```diff
@@ -99,6 +110,7 @@
      */
     private Link link;
 
+
```

Comment:
I don't think we need to add this space.

Reply:
done here : efcc730cabab80004a7d621eb002582a8b567b45